src/ethereum/hardforks.py

```diff
@@ -0,0 +1,46 @@
+"""
+Hardforks
+^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Lists of hardfork blocks for Ethereum and its various testnets.
+"""
+
+from ethereum import (
+    berlin,
+    byzantium,
+    constantinople,
+    dao_fork,
+    frontier,
+    homestead,
+    istanbul,
+    muir_glacier,
+    spurious_dragon,
+    tangerine_whistle,
+)
+
+mainnet = {
+    1: frontier,
+    1150000: homestead,
+    1920000: dao_fork,
+    2463000: tangerine_whistle,
+    2675000: spurious_dragon,
+    4370000: byzantium,
+    7280000: constantinople,
+    9069000: istanbul,
+    9200000: muir_glacier,
+    12244000: berlin,
+}
+
+goerli = {
+    1: constantinople,
+    1561651: istanbul,
+    4460644: berlin,
+    #    5062605: london,
+}
```

Comment:
Hm, how do you feel about having testnets visible in the rendered documentation? Personally, I think we should stick to purely documenting mainnet (but if we can sync other networks, that's a bonus.) Thoughts?